pkg/infra/usagestats/statscollector/service.go

```diff
@@ -157,7 +157,6 @@ func (s *Service) collectSystemStats(ctx context.Context) (map[string]interface{
 	m["stats.api_keys.count"] = statsResult.APIKeys
 	m["stats.data_keys.count"] = statsResult.DataKeys
 	m["stats.active_data_keys.count"] = statsResult.ActiveDataKeys
-	m["stats.public_dashboards.count"] = statsResult.PublicDashboards
```

Comment:
my bad here. You're right. So, I think it would be good to have 2 metrics in this case. This one for stats which gives you the total amount of public dashboards (I should change the db query, because it has an incorrect _where_ clause) and then another metric with many labels (enabled/disabled, email/public) for cloud. 
Do you agree?